doc/en/components/grids/_shared/row-pinning.md

```diff
@@ -110,9 +108,10 @@ A row is pinned below the last pinned row. Changing the order of the pinned rows
 <{ComponentSelector} id="grid1" auto-generate="true">
 </{ComponentSelector}>
 ```
-```ts
+
+```typescript
 constructor() {
-    var grid1 = document.getElementById('grid1') as IgcGridComponent;
+    var grid1 = document.getElementById('grid1') as {ComponentName};
```

Comment:
Needs to end with `Component`, for example: {ComponentName}Component, since otherwise it would be just IgcGrid/IgcTreeGrid which is incorrect type.